### PR TITLE
Refactor to TF2

### PR DESCRIPTION
**Pull request by rodrigoalmeida94**
- [x] Update patches file
- [x] Load model from file (placeholders for now)

### CODE REVIEW

**Review by markusuwe — COMMENTED** [review]
Nice. Looks much better than before.

File: src/patches.py [hunk 1]
```diff
@@ -251,352 +173,71 @@ def get_test_patches60(
...
         y_tiles = int(ceil(size[0] / float(patch_size)))
+        # print('Tiles per image {} {}'.format(x_tiles, y_tiles))
 
         # Initialize image
-        images = np.zeros((a_re.shape[1], size[0], size[1])).astype(np.float32)
+        # print('Image size is: {}'.format(size))
+        images = np.zeros((a.shape[1], size[0], size[1])).astype(np.float32)
 
-        LOGGER.info(images.shape)
+        print(images.shape)
```
[the remark this answers is not in the document]
Comment (rodrigoalmeida94) [reply src/patches.py]: removed... unnecessary print statement

File: src/patches.py [hunk 2]
```diff
@@ -1,130 +1,106 @@
-"""
```
[the remark this answers is not in the document]
Comment (rodrigoalmeida94) [reply src/patches.py:1]: Fixed, thanks!

**Comment by nekhtiari** [comment]
Nice, LGTM

**Pull request merged**